CHANGELOG.md

```diff
@@ -6,14 +6,15 @@ to make them as straightforward as possible.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project
 adheres to [Semantic Versioning](http://semver.org/).
 
-## [1.0.0] - In Progress
+## [1.0.0rc1] - In Progress
```

Comment:
rc1?

Reply:
Release candidate 1 :)

Reply:
Strictly speaking according to Semver it sould be 1.0.0-rc1. Why not call it 1.0.0-alpha, see Semver  spec 9: https://semver.org/#spec-item-9

Reply:
https://www.python.org/dev/peps/pep-0440/#pre-releases

I followed pypip guide PEP 440 :)

Reply:
For me it's just a format, it can be any one.

Reply:
the reason of not being alpha because i do not know what will be the name after alpha, beta

Reply:
But if we are following Semver according to the docs as it says in the changelog, it should be 1.0.0-rc.1? So either we change semver to pep440 or we follow Semver.

Reply:
checking https://github.com/tensorflow/tensorflow/tags so let's go for your suggestion